presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -691,6 +692,10 @@ std::vector<std::string> PrestoServer::registerConnectors(
           fileName.substr(0, fileName.size() - kPropertiesExtension.size());
 
       auto connectorConf = util::readConfig(entry.path());
+      // Store catalogName for use by filesystem to uniquely identify
```

Comment:
Perhaps `Store catalogName as 'connector.id' to uniquely identify a connector instance`?
Multiline comments also need `///`